.storybook/setupTests.js

```diff
@@ -2,6 +2,8 @@ import React from 'react';
 import Enzyme, {shallow, render, mount} from 'enzyme';
 import Adapter from 'enzyme-adapter-react-16';
 import * as rxjs from 'rxjs';
+import 'core-js/stable';
```

Comment:
what are these for?

Reply:
maybe a short comment.

Reply:
They are [polyfills](https://babeljs.io/docs/en/next/babel-polyfill.html) so we can use `async`/`await`.
I changed implementation to observable, so this will not be necessary anymore.